shepherd_bms/src/analyzer.cpp

```diff
@@ -194,6 +194,7 @@ void Analyzer::calcContCCL()
 
 void Analyzer::calcOpenCellVoltage()
 {
+    StateMachine StateMachine;
```

Comment:
uh we def should not reinstate the state machine, there should only be one state machine. It might be better to use the segment interface to find this out or leverage the bmsdata struct